service/src/main/resources/db/changesets/20250225_add_quota_units.yaml

```diff
@@ -0,0 +1,28 @@
+# add quota_units field to pipeline_quotas table and set value for array_imputation pipeline
+
+databaseChangeLog:
+  -  changeSet:
+       id:  add and populate quota_units column in pipeline_quotas table
+       author:  mma
+       changes:
+         # add quota_units column to pipeline_quotas table
+         - addColumn:
+             tableName: pipeline_quotas
+             columns:
+               - column:
+                   name: quota_units
+                   type: text
+                   constraints:
+                     nullable: true
+         # update quota_units value for array_imputation pipeline
+         - update:
+             tableName: pipeline_quotas
+             columns:
+               - column:
+                  name: quota_units
+                  value: 'samples'
```

Comment:
should we update the e2e to test for this too?  prob not necessary

Reply:
we don't currently check all the fields returned from the quota endpoint... i could make the change to ask for all of them, but currently we only check the value of `response['quotaConsumed']`